doc/configuration_options.rst

```diff
@@ -431,6 +431,9 @@ The following options are valid in version ``1`` (beside the generic options):
 
 * ``jenkins_job_timeout``: the job timeout for *CI* jobs.
 
+* ``jenkins_job_upstream_trigger``: name(s) of other CI job(s) which, when
+  completed, should trigger this job to be built.
```

Comment:
The term "completed" is a bit fuzzy here. It would be good to clarify what build status is required to trigger the downstream job. I guess *stable* as well as *unstable* is sufficient?